Kiss.Bff.EndToEndTest/NieuwsEnWerkInstructies/NieuwsEnWerkInstructiesScenarios.cs

```diff
@@ -571,111 +572,225 @@ public async Task Scenario18()
+             
         await Step("And confirms a pop-up window with the message ‘Weet u zeker dat u dit bericht wilt verwijderen?’");
 
+        var deleteButton = nieuwsRow.GetByTitle("Verwijder").First;
+
+        using (var _ = Page.AcceptAllDialogs())
+        {
+            await deleteButton.ClickAsync();
+        }
+
         await Step("Then the nieuwsbericht is no longer in the list");
 
-        Assert.Inconclusive("Not implemented yet");
+        var deletedRow = Page.GetBeheerRowByValue(nieuws.Title); 
+
+        await Expect(deletedRow).ToBeHiddenAsync();
+
+
     }
 
     [TestMethod]
     public async Task Scenario19()
     {
         await Step("Given there is at least 1 werkinstructie");
 
+        var werkinstructie = await Page.CreateBericht(new() { Title = Guid.NewGuid().ToString(), BerichtType = BerichtType.Werkinstructie });
+
         await Step("And the user is on the Nieuws and werkinstructiesscreen available under Beheer");
 
+        await Page.NavigateToNieuwsWerkinstructiesBeheer();
+
         await Step("When user clicks on the delete icon of the werkinstructie in the list");
 
+        var werkinstructieRow = Page.GetBeheerRowByValue(werkinstructie.Title);
+
         await Step("And confirms a pop-up window with the message ‘Weet u zeker dat u dit bericht wilt verwijderen?’");
 
+        var deleteButton = werkinstructieRow.GetByTitle("Verwijder").First;
+
+        using (var _ = Page.AcceptAllDialogs())
+        {
+            await deleteButton.ClickAsync();
+        }
+
         await Step("Then the werkinstructie is no longer in the list");
+         
+        await Expect(Page.GetBeheerRowByValue(werkinstructie.Title)).ToBeHiddenAsync();
 
-        Assert.Inconclusive("Not implemented yet");
     }
 
     [TestMethod]
     public async Task Scenario20()
     {
         await Step("Given there is at least 1 nieuwsbericht");
 
+        await using var skill = await Page.CreateSkill(Guid.NewGuid().ToString());
+        await using var nieuw = await Page.CreateBericht(new() { Title = Guid.NewGuid().ToString(), BerichtType = BerichtType.Nieuws, Skill=skill.Naam , Body= Guid.NewGuid().ToString()});
+
         await Step("And the user is on the Nieuws and werkinstructiesscreen available under Beheer");
 
+        await Page.NavigateToNieuwsWerkinstructiesBeheer();
+
         await Step("When the user clicks on the arrow button of the nieuwsbericht");
 
-        await Step("Then the Type, Titel, Inhoud, Publicatiedatum, Publicatie-einddatum and Skills of the nieuwsbericht are visible in a details screen");
+        await Page.GetBeheerRowByValue(nieuw.Title).GetByRole(AriaRole.Link).ClickAsync();
 
-        Assert.Inconclusive("Not implemented yet");
+        await Step("Then the Type, Titel, Inhoud, Publicatiedatum, Publicatie-einddatum and Skills of the nieuwsbericht are visible in a details screen");
+        
+        await Expect(Page.Locator("#titel")).ToHaveValueAsync(nieuw.Title);
```

Comment:
The same goes for these locators: if we need 'custom' locators like this (by id, using the `+` sign), maybe the content is not accessible enough. Let's document these findings in the readme.

Reply:
accessible to who? to the test or to visually disabled users? An Id seems pretty accessible to me for an automated test. What would be a better solution?

Reply:
Accessible to users with an impairment or users who navigate using the keyboard, any user basically.

Users don't identify an input field using an id. Most likely they'll identify it using the label. If it is not possible to identify an input field based on its label, that's an accessibility issue that I think we should fix. If it is possible to identify an input field based on it's label, I would prefer we do that in the test as well.

Users also don't identify divs. The fact that we use `label:text('Inhoud') + div` as a locator indicates that there is no better way to identify the content of the rich text editor. That's another issue I think we should fix.

Reply:
the problem here is that the html is controlled by a third party component. it could change with an update and break the test. I think we should avoid that

Reply:
Understood. Let's leave the rich text editor locator the way it is now. I do think we should be able to locate the `Titel` input field by its label.